WBooks/Book Comments/BookCommentsViewController.swift

```diff
@@ -43,23 +46,22 @@ class BookCommentsViewController: UIViewController {
     func onCommentLoadSuccess(comments: [Comment]) {
         _viewModel.onCommentLoadSuccess(comments: comments)
         _view.stopActivityIndicator()
-        if _viewModel.comments.isEmpty {
+        if _viewModel.comments.value.isEmpty {
```

Comment:
you could manage this in the comments binding you set before like: 
`.startWithValues { [weak self] comments in
  if comments.isEmpy { self?._view.displatyNoCommentsYet() }
  else { self?._view.comments.reloadData() }`